src/ol/source/Cluster.js

```diff
@@ -132,7 +140,23 @@ class Cluster extends VectorSource {
   }
 
   /**
-   * handle the source changing
+   * Replace the wrapped source.
+   * @param {VectorSource|null} source The new source for this instance.
```

Comment:
`null` is implicitly allowed, so no need to add it to the type.

Reply:
I added null because it didn't show it as an allowed value in the apidoc otherwise.

Reply:
There is no need to document that here, but maybe it should be documented somewhere else that `null` can be used to unset things, because that works everywhere in OpenLayers.